REUSE.toml

```diff
@@ -158,6 +158,8 @@ SPDX-License-Identifier = "LicenseRef-qskinny"
 path = [
   "tools/lsp/ui/assets/chevron-down.svg",
   "tools/lsp/ui/assets/inspect.svg",
+  "tools/lsp/ui/assets/search.svg",
```

Comment:
Please keep this ordered... that way you will notice that `search.svg` is already listed.